composer.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "netzstrategen/image-disclaimer",
+    "type": "wordpress-plugin",
+    "authors": [
+        {
+            "name": "netzstrategen",
+            "email": "hallo@netzstrategen.com",
+            "homepage": "https://netzstrategen.com"
+        }
+    ],
+    "dist": {
+        "url": "git@github.com:netzstrategen/woocommerce-image-disclaimer.git",
+        "type": "git"
+    },
```

Comment:
dist must not be specified statically.

Reply:
dist is still contained.

Reply:
Ah sorry, I wrongly understood you mean to remove only the `git` type. I fixed this into a new fixup commit.